WordPress/src/main/java/org/wordpress/android/ui/SlidingScrollView.java

```diff
@@ -0,0 +1,51 @@
+package org.wordpress.android.ui;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.view.ViewTreeObserver;
+import android.widget.ScrollView;
+
+public class SlidingScrollView extends ScrollView {
```

Comment:
Not sure about pointing to `SlidingRelativeLayout` but, lemme add a small comment about the need for `SlidingScrollView` in the first place: So, the new class is needed in order to support proper scrolling up/down when the IME is open, when the user types in the `EditText`s. The whole view is rather tall and some is hidden when the IME opens. The `SlidingRelativeLayout `makes it harder to support the scrolling.